app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -40,6 +48,22 @@
     ))
 }
 
+@noRelatedMaterial = {
+  <li>None</li>
+}
+
+@relatedMaterialLink(link: RelatedMaterial.LinkOnly) = {
+  <li><a class="relatedMaterialLink" href="@link.linkHref">@link.linkText</a></li>
```

Comment:
Can we please use all lower case attribute names within the HTML, so this should be `<a class="related-material-link"..>` as this is a W3C recommendation and it is how attributes are shown within the GOV.UK Design System.

Reply:
That's good to know.